packages/web-components/src/components/va-button/va-button.tsx

```diff
@@ -128,11 +128,13 @@ export class VaButton {
     }
   };
 
+  // button text; custom text takes precedence
   private getButtonText = (): string => {
+    if (this.text) return this.text;
     if (this.continue) return 'Continue';
     if (this.back) return 'Back';
     if (this.loading && !this.text) return 'Loading...';
-    return this.text;
+    return '';
```

Comment:
Do we think this change will create any issues in vets-website? The update is for custom text (`this.text`) to always take precedence over the variation text. So someone could adjust the "Continue" or "Back" strings if they'd like. 

Previously, `this.text` would be the final default return. Now the default is an empty string (`''`).